src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -737,10 +737,12 @@ viewInlineTag { showTagsInline, isInteractive, maybeTool } highlightable =
                         , Css.boxShadow5 Css.zero (Css.px 1) (Css.px 1) Css.zero Colors.gray75
                         , Css.display Css.none
                         , if showTagsInline then
-                            Css.display Css.inline |> Css.important
+                            Css.batch [ Css.display Css.inline |> Css.important, MediaQuery.highContrastMode [ Css.property "forced-color-adjust" "none", Css.color Colors.black |> Css.important ] ]
```

Comment:
@charbelrami is there a system color we could use instead of black?

Reply:
replaced with `color: initial`

Reply:
I haven't used `Css.property "forced-color-adjust" "none"` before, do we still need it?

Reply:
Yes, otherwise the user agent on Windows will force its own colors which results in inline tags with dark foreground on dark background and we would not be able to change them.